dts/arm/nxp/nxp_k6x.dtsi

```diff
@@ -292,22 +292,52 @@
 			prescaler = <2>;
 		};
 
-		pwm0: pwm@40038000{
-			compatible = "nxp,k64f-pwm";
+		ftm0: ftm@40038000{
+			compatible = "nxp,kinetis-ftm";
 			reg = <0x40038000 0x98>;
+			interrupts = <42 0>;
+			label = "PWM_0";
 			prescaler = <2>;
 			period = <1000>;
 			clock-source = <0>;
 			/* channel information needed - fixme */
+			status = "disabled";
 		};
 
-		pwm1: pwm@40039000{
-			compatible = "nxp,k64f-pwm";
+		ftm1: ftm@40039000{
+			compatible = "nxp,kinetis-ftm";
 			reg = <0x40039000 0x98>;
+			interrupts = <43 0>;
+			label = "PWM_1";
+			prescaler = <2>;
```

Comment:
I was on the fence about leaving vs. removing them since @agross-linaro originally put them in as placeholders, and eventually they should probably be used. I didn't implement them now because [ZEP-2018](https://jira.zephyrproject.org/browse/ZEP-2018) is held up by this PR.

On a related note, do you have an opinion about the dts node name? e.g., 
`pwm0: pwm@40038000` vs. `ftm0: ftm@40038000`? pwm is the name of the Zephyr interface, whereas ftm is the name of the hardware IP.